### PR TITLE
Fix readonly field

### PR DESCRIPTION
**Pull request by haslinghuis**
Adds grey background color to readonly field in motors tab for idleMinRpm

image

image

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Maybe use same method we use in PID Tuning to set the display characteristics of PID values that are controlled by sliders?  They use a consistent way to indicate the 'display-only' state that perhaps we could use here?

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
